mod.ts

```diff
@@ -49,14 +49,14 @@ export interface Routes {
  * The route handler declared for `404` will be used to serve all
  * requests that do not have a route handler declared.
  */
-export function serve(userRoutes: Routes): void {
+export function serve(userRoutes: Routes, addr = ":8000"): void {
```

Comment:
two thoughts: 

1. is the `:` necessary here? It may be easier simply to make it a `number` and then do `localhost:${port}`
2. What about turning this into a type:
```
type SiftOptions = {
   port: number
}
```

It may be overkill now, but it saves having to deal with juggling multiple types of arguments later.

Reply:
We can use the https://doc.deno.land/deno/stable/~/Deno.ListenOptions interface here.